lib/galaxy/tool_util/verify/__init__.py

```diff
@@ -131,10 +131,12 @@ def get_filename(filename):
             elif compare == 're_match_multiline':
                 files_re_match_multiline(local_name, temp_name, attributes=attributes)
             elif compare == 'sim_size':
-                delta = attributes.get('delta', '100')
+                delta = float(attributes.get('delta', '10000'))
```

Comment:
We shouldn't increase the default delta, that'll alter existing tools' test results

Reply:
True, but the default used here was just inconsistent with:  https://github.com/galaxyproject/galaxy/blob/5214ba0b5b1af85ec32973656e9b73fe0e9af7f4/lib/galaxy/tool_util/parser/xml.py#L615

Reply:
We should have a "constant" for that.

Reply:
Is there already a place where such constants used for parsing the xml are stored?

Reply:
I don't think so, but maybe `lib/galaxy/tool_util/parser/util.py` would be a good start ?